docs/releases/index.rst

```diff
@@ -11,34 +12,64 @@ Also be aware of the :ref:`important changes in the Translate Toolkit
 If you are upgrading Pootle, you might want to see some tips to ensure your
 :doc:`upgrade goes smoothly </server/upgrading>`.
 
+Final releases
+==============
+
+2.8 release
+-----------
+.. toctree::
+   :maxdepth: 1
+
+   2.8.0
```

Comment:
Note this is listed under _Final releases_, but it's not a final release. I see there's a warning in the _2.8.0_ doc's title, however that might not be enough to prevent confusion. I believe putting it under a _Upcoming releases_ or similar heading might help in that regards.